addons/l10n_lu/data/account.tax.template-2015.csv

```diff
@@ -405,4 +405,6 @@ lu_2011_tax_VP-PA-3,Vente Prestations 3% - Pays,VP-PA-3,sale,percent,3,lu_2011_a
 lu_2011_tax_VP-PA-6,Vente Prestations 6% - Pays,VP-PA-6,sale,percent,6,lu_2011_account_4614111,lu_2011_account_4614111,lu_2011_chart_1,504,,"tag_lu_200,tag_lu_278"
 lu_2015_tax_VP-PA-8,Vente Prestations 8% - Pays,VP-PA-8,sale,percent,8,lu_2011_account_4614111,lu_2011_account_4614111,lu_2011_chart_1,505,,"tag_lu_294,tag_lu_218"
 lu_2015_tax_SANS,Sans Taxes,SANS,purchase,percent,0,,,lu_2011_chart_1,506,,tag_lu_598
-lu_2015_tax_SANS,Sans Taxes,SANS,sale,percent,0,,,lu_2011_chart_1,507,,tag_lu_598
\ No newline at end of file
+lu_2015_tax_SANS,Sans Taxes,SANS,sale,percent,0,,,lu_2011_chart_1,507,,tag_lu_598
```

Comment:
Well actually you don't fix the bug as there is still twice the same row.  If they have the same id, the install will update the first line with the second one. So, in the end, we get only the sale line.
I agree we should keep it for backward compatibility (even if a tax template is probably not of much use) but you can remove the purchase line then.

Reply:
For this specific case I don't think backward compatibility is really important. So I propose to remove the bad one.